src/core/public/index.ts

```diff
@@ -170,7 +170,6 @@ export type {
 export { CoreScopedHistory } from '@kbn/core-application-browser-internal';
 
 export type {
-  SavedObjectsClientContract,
```

Comment:
The PR also removes some exported members from `src/core/public/index.ts` and `src/plugins/saved_objects/public/saved_object/index.ts`.
Please limit the changes to what's strictly in the scope of the PR.

Reply:
the cahnges in src/plugins/saved_objects/public/saved_object/index.ts were made to original state by creating new commit instead of revert is it ok @gsoldevila

Reply:
Yep, that's fine, commits are squashed on merge.
The changes in `src/core/public/index.ts` still need to be reverted though.